src/test/java/com/google/gcp_variant_transforms/entity/VariantTest.java

```diff
@@ -0,0 +1,60 @@
+// Copyright 2020 Google LLC
+
+package com.google.gcp_variant_transforms.entity;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
```

Comment:
Same here.

Reply:
I think import Mockito to mock VariantContext object is needed. Other issued are solved.

Reply:
No, I meant please import individual classes rather than using *, for Mockito and Truth imports. Seems like I missed this in the other PR, so we should modify that file at some point (not here).

Reply:
Resolved.